lib/viewmodels/event.list.viewmodel.dart

```diff
@@ -93,7 +93,7 @@ class EventListViewModel extends ChangeNotifier {
 
     _events = [];
     for (EventModel eM in eventModels) {
-      if (eM.startDateAndTime?.isAfter(startOfToday) ?? false) {
+      if (eM.endDate?.isAfter(startOfToday) ?? false) {
```

Comment:
Haben Veranstaltung immer ein "generiertes" Enddatum? Oder kann es sein, dass hier Veranstaltungen nicht aufgenommen werden, die kein Enddatum haben, aber das Startdatum noch bevorsteht?

Reply:
Das Enddate ist immer generiert, die MiDa schickt startDateAndTime und die durationDays (bei Tagesveranstaltungen 1, bei der Diko z.B. 3)

Hab aber gerade gesehen dass da noch eine Klammer bei dem Optional fehlt (aktuell geht die Diko in der App bis zum 08.), mache noch einen Commit